Week/index.ts

```diff
@@ -0,0 +1,54 @@
+import { DayOfWeek } from "DayOfWeek"
+import { isly } from "isly"
+import { Date } from "../Date"
+import { Number as WeekNumber } from "./Number"
+import { Parts as WeekParts } from "./Parts"
+
+export type Week = `${number}-W${number}`
```

Comment:
Something like this?
```ts
`${Year.Numeric}-${Week.Numeric}`
```

Reply:
The type then becomes to complex for Typescript to handle.